src/main/java/com/knighttodo/knighttodo/service/impl/TodoBlockServiceImpl.java

```diff
@@ -35,7 +35,7 @@ public TodoBlockVO save(TodoBlockVO todoBlockVO) {
     public List<TodoBlockVO> findAll() {
         return todoBlockGateway.findAll()
             .stream()
-            .map(todoBlockMapper::toTodoBlockVO)
+            .map(todoBlock -> todoBlockMapper.toTodoBlockVO(todoBlock))
```

Comment:
Why do you decide to use lambda instead of a method reference?

Reply:
My bad, it is left over from attempts to avoid looping using context